src/internal/operations/manifests.go

```diff
@@ -163,7 +163,7 @@ func verifyDistinctBases(ctx context.Context, mans []*kopia.ManifestEntry, errs
 			if b, ok := reasons[reasonKey]; ok {
 				failed = true
 
-				et.Add(clues.New("manifests have overlapping reasons").
+				el.AddRecoverable(clues.New("manifests have overlapping reasons").
```

Comment:
this should really be a failure as it's not clear what base we should be sourcing items from

Reply:
Done